src/containers/CreateArticle/ArticleTitle.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Editor } from "@tinymce/tinymce-react";
+import PropTypes from "prop-types";
+
+const apiKey = "0naso5vy7fw5eu2eqblor00o1c326x0hupermg4cli0rqoqn";
```

Comment:
You should always values like on line 5 in the env.

Reply:
@emmaadesile resolve this.

Reply:
I have resolved this